README.md

```diff
@@ -20,10 +20,10 @@ To update to a new version, simply repeat these steps.
 Unfortunately, apps can't declare new mimetypes on the fly. To make
 Keeweb work properly, you need to add a new mimetype in the
 `mimetypemapping.json` file (also see the Nextcloud manual at
-https://docs.nextcloud.com/server/14/admin_manual/configuration_mimetypes/index.html).
+https://docs.nextcloud.com/server/16/admin_manual/configuration_mimetypes/index.html).
```

Comment:
should be `stable` to point always to the latest documentation

also refer to 
https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#maintenance-commands